cloudflare.go

```diff
@@ -6,6 +6,7 @@ import (
 	"encoding/json"
 	"io"
 	"io/ioutil"
+	"log"
```

Comment:
I completely agree -- this was left over from my debugging in terraform. I'll get rid of it as I finish adding some more functions and tests. Thanks for the quick feedback!